### DISCUSSION TITLE
The problem

### DISCUSSION THREAD

**Discussion by Polivio781** [home-assistant/frontend#23453] [Configuration] [answered]
Merry Christmas, my problem related to the processor does not improve despite having performed the last update, as soon as I enter the app on iphone or ipad the processor starts working from 8% up to 94% the problem is that if I leave the app in the processor continues to work a lot no longer on how to do the whole thing runs on raspberry pi4 with 8 gb of ram in case I can switch to a raspberry pi 5 if necessary, but I would like confirmation so as not to spend money unnecessarily

**Comment by ildar170975** [comment]
Consider using punctuation marks & give a normal title.

**Comment by Polivio781** [comment]
ok Problem with Processor Home Assistant on rapberry pi 4

**Comment by ribetm** [comment, marked as answer]
If you're experiencing high processor usage on your server, even when the Web UI is closed, then it's likely related to the core and not the frontend. HA core is in [another repo](https://github.com/home-assistant/core/issues/new/choose) and you should report such issues there.

Anyway, a Pi5 is not required. For reference, my instance stays at 0% even with the Android app and web UI open. You need to provide more information to diagnose this issue. The link I provided should guide you through providing the maintainers with the minimum required information.

You should also confirm that:
- you are talking about your Pi processor and not your iPad's.
- this processor usage is actually coming from HA and not some other app you may have installed
